lib/rules/accessor-pairs.js

```diff
@@ -63,3 +63,11 @@ module.exports = function(context) {
     };
 
 };
+
+module.exports.documentation = {
+    errors: ["Getter is not present", "Setter is not present"],
+    description: "enforces getter/setter pairs in objects",
+    ruleType: "Best Practices",
+    recommended: false,
+    default: [{getWithoutSet: false, setWithoutGet: true}]
```

Comment:
Hmmm.. What if we were to change the way options are accessible in the rules. Schema would parse an array and create an object with all options available, then inside the rule, instead of doing ```var getWithoutSet = context.options[0]``` you would be able to do ```context.options.getWithoutSet```?